app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var run = require('./cli/lib/gateway')();
+const os = require('os');
+
+const options = {};
+
+options.env = process.env.EDGEMICRO_ENV;
+options.key = process.env.EDGEMICRO_KEY;
+options.secret = process.env.EDGEMICRO_SECRET;
+options.org = process.env.EDGEMICRO_ORG;
+options.configDir = process.env.EDGEMICRO_CONFIG_DIR;
+options.processes = os.cpus().length;
+options.port = process.env.PORT || 8000;
+
+run.start(options);
```

Comment:
Would possibly exposing this part of the module for consumption, and then making this script a separate package make any sense?